scripts/import_missing_qc_variants_files.py

```diff
@@ -0,0 +1,124 @@
+import os
+import csv
+import subprocess
+from tqdm import tqdm
+import time
+
+# file constant
+folder_path = '/mnt/disks/data/data-directory/variant_qc/panel_variant_qc'
+output_fname = folder_path + "/combined_sorted_output_file.tsv"
+# Output tsv Gzipped file path
+output_gzipped_tsv = output_fname + ".gz"
+skipped_file_name = 'chrX.removed.add_LCR.add_XPAR.add_failed_filter_awk.tsv'
```

Comment:
Hardcoded arguments are not that useful in scripts meant to be reused, you could use [argparse](https://docs.python.org/3/library/argparse.html) to get a simple argument selection going.
Also, why is chrX skipped? We would most likely want to have that included in the data